src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -972,13 +995,13 @@ static DistributeObjectOps TextSearchConfig_AlterOwner = {
 	.markDistributed = false,
 };
 static DistributeObjectOps TextSearchConfig_Comment = {
-	.deparse = DeparseTextSearchConfigurationCommentStmt,
+	.deparse = DeparseCommentStmt,
 	.qualify = QualifyTextSearchConfigurationCommentStmt,
```

Comment:
Let's add a reminder comment here that we should refactor this when we add more comment types:
```suggestion
    /* TODO: When adding new comment types we should create an abstracted qualify function, just like we have an abstract deparse and adress function */
	.qualify = QualifyTextSearchConfigurationCommentStmt,
```

Let's add such a comment to both TextSearchConfig_Comment and TextSearchDict_Comment, so we don't forget.

It doesn't make sense to that in this PR though, since we don't require qualification for Database and Role

Reply:
Fixed